frontend/src/pages/sites/sites.tsx

```diff
@@ -80,21 +89,25 @@ const Sites: React.FC = () => {
     },
   );
 
+  const showSites = useMemo(() => {
```

Comment:
```suggestion
  const hasSites = useMemo(() => {
```

Reply:
We already have the hasSites variable, it shows the result of checking for the presence of sites in the array of sites after the fetch.

Reply:
okay, should be a different name. `show` is a noun. boolean variables should start with `yes`. Ex `has`, `is`, etc

Reply:
ready